libs/build_yafyaml.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eux
+
+name="yafyaml"
+repo=${1:-${STACK_yafyaml_repo:-"Goddard-Fortran-Ecosystem"}}
```

Comment:
same comment as above.

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884